inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -170,7 +170,8 @@ MKLDNNNode::MKLDNNNode(const InferenceEngine::CNNLayerPtr& layer, const mkldnn::
         if (!(CaselessEq<std::string>()(layer->type, "memory") ||
             CaselessEq<std::string>()(layer->type, "memoryinput") ||
             CaselessEq<std::string>()(layer->type, "output") ||
-            CaselessEq<std::string>()(layer->type, "reorder"))) {
+            CaselessEq<std::string>()(layer->type, "reorder") ||
+            CaselessEq<std::string>()(layer->type, "convert"))) {
```

Comment:
If I understand correctly, these checks are necessary to allow only specific nodes to be instantiated with default CNN layer. The reorder type when it is to be inserted during the graph initialization is an example. Since in this PR we need to instantiate the convert node also from default CNN layer, we need to add it in this check list.